script.js

```diff
@@ -11,26 +11,22 @@ for (let btn of readMoreBtns) {
 // toggle forms
 const contactForm = document.querySelector('.contact__form');
 const orderForm = document.querySelector('.order__form');
-let formSelectors = document.querySelectorAll('.selector__button');
+const formSelectors = document.querySelectorAll('.selector__button');
 
-formSelectors.forEach(button => {
+formSelectors.forEach((button) => {
   button.addEventListener('click', () => {
-    if(button.value === 'contactForm'){
-      contactForm.classList.add('active__form');
-      orderForm.classList.remove('active__form');
-    }else{
-      contactForm.classList.remove('active__form');
-      orderForm.classList.add('active__form');
-    }
-  })
-})
```

Comment:
We need this. If you open the page now and click the radio buttons you can see that the forms will change even if you click a button that was already selected. The function needs to determine if you are choosing a form that is already active so it doesn't act erratically.

Reply:
Fixed it.